pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -5233,42 +5241,41 @@ protected void internalGetReplicatedSubscriptionStatus(AsyncResponse asyncRespon
         });
     }
 
+    private CompletableFuture<Map<String, Boolean>> getReplicatedSubscriptionStatusFromBroker(TopicName partitionName,
```

Comment:
`getReplicatedSubscriptionStatusFromBroker` --> `getReplicatedSubscriptionStatusFromLocalBroker` or `getLocalReplicatedSubscriptionStatus` ?

Reply:
Update to `getReplicatedSubscriptionStatusFromLocalBroker `